packages/zipkin/src/tracer/record.js

```diff
@@ -0,0 +1,13 @@
+class Record {
```

Comment:
for types that are basically the same as finagle, should we link? or cover in overall readme that this is largely inspired by finagle's Tracer